shuup/core/models/_orders.py

```diff
@@ -407,6 +407,12 @@ def _set_partially_paid(self):
     def is_paid(self):
         return (self.payment_status == PaymentStatus.FULLY_PAID)
 
+    def is_not_paid(self):
+        return (self.payment_status == PaymentStatus.NOT_PAID)
+
+    def is_zero_price(self):
+        return (self.taxful_total_price.__eq__(TaxfulPrice(0, self.shop.currency)))
```

Comment:
(you aren't supposed to call `__eq__` directly anyway... it would have been `self.taxful_price == TaxfulPrice(0, self.currency)`, but using `.value` is better here)